charts/alfresco-connector-msteams/tests/configmap_test.yaml

```diff
@@ -0,0 +1,33 @@
+suite: test msteams-connector configmap
+templates:
+- config-connector-msteams.yaml
+- configmap-repository.yaml
```

Comment:
I prefer we keep unrelated resources in separated test suite files, e.g. config-connector-msteams_test.yaml and configmap-repository_test.yaml

in that way you can also avoid having to repeat `template` in every test